y/error_test.go

```diff
@@ -0,0 +1,27 @@
+package y
+
+import (
+	"errors"
+	"github.com/stretchr/testify/require"
```

Comment:
this import order has to look like this:

"errors"
"testing"

"github.com/stretchr/testify/require"

Reply:
Why?

Reply:
Strange rule to place in lint. I am not even sure what is the value add of the rule.

Reply:
done

Reply:
We do it to separate standard lib imports from external imports. I have always done this for as long as I have been writing Go code. It is a lot nicer and easier to read this way I believe.

Reply:
Ok, I let goland do this for me.